hello.py

```diff
@@ -1 +1 @@
-print("Hello")
+print("Hello!")
```

Comment:
Great code!

It would be better with 3 "!"
```suggestion
print("Hello !!!")
```